src/molecules/user/user.js

```diff
@@ -1,19 +1,31 @@
-import React from 'react';
-import Avatar from 'src/atoms/avatar';
-import Highlight from 'src/atoms/highlight';
-import { propTypes, defaultProps } from './prop-type';
-import UserMeta from './user-meta';
+import React from "react";
+import UserMeta from "./user-meta";
+import Avatar from "src/atoms/avatar";
+import Highlight from "src/atoms/highlight";
+import { propTypes, defaultProps } from "./prop-type";
+import { mentionLink } from "./util";
 import {
 	UserContainer,
 	UserAvatar,
 	UserName,
 	UserDescription,
-	UserDescriptionHighlight,
-} from './elements';
+	UserDescriptionHighlight
+} from "./elements";
 
 const decorators = {
-	'@': (segment, match, key) => <UserDescriptionHighlight key={key}>{segment}</UserDescriptionHighlight>,
-	'#': (segment, match, key) => <UserDescriptionHighlight key={key}>{match}</UserDescriptionHighlight>,
+	"@": (segment, match, key) => (
+		<UserDescriptionHighlight
+			key={key}
+			href={mentionLink(segment)}
+			target="_blank"
+			rel="noopener noreferrer"
```

Comment:
I always forget the `rel` attributes on these things 😄 Thanks! I think the `noopener` is a safe one, but I'm not sure about the `noreferrer`. Do you have any thoughts in this one?